pyccel/ast/numpyext.py

```diff
@@ -173,6 +176,66 @@ def _sympystr(self, printer):
     def arg(self):
         return self._arg
 
+#==============================================================================
+class NumpyArange(NumpyNewArray):
+    """
+    Represents a call to  numpy.arange for code generation.
+
+    Parameters
+    ----------
+        stop : Number
+            End of interval.
+
+        start : Number
+            Start of interval, default value 0
+
+        step : Number default 1
+            Spacing between values, default value 1
+
+        dtype : Datatype
+            The type of the output array
```

Comment:
I have a few comments about this docstring:

- Please reorder the parameters with `start` before `stop`.

- Also, there is no need to indent them:
```
Parameters
----------
param : type
    Description.
```
- Is `Number` really a type?

- Can you be a bit more consistent in the way that you give the default value?

Reply:
Can you be a bit more consistent in the way that you give the default values?
can you explain more what do you mean by this

Reply:
For `start` you write the default value once, for `step` you write it twice, for `dtype` you don't write it at all.

Reply:
ah true sorry